schema/dataKeys.json

```diff
@@ -319,7 +317,7 @@
           "type": "array",
           "items": {
             "type": "object",
-            "additionalProperties": false,
+            "additionalProperties": true,
```

Comment:
This doesn't seem right. We don't want to allow any other fields other than the ones listed here.

Reply:
We have a problem with how one of the E2E  pipelines is defining source https://github.com/konflux-ci/e2e-tests/blob/b2ed0c117cb9ee726de051a3636f29e00eef6b7d/tests/release/pipelines/multiarch_advisories.go#L238C1-L242C8

So, it's just going to be a TMP change. Want see if we can pass e2e, and then we'll update depending on which one is correct.